hrms/hr/doctype/shift_type/shift_type.py

```diff
@@ -242,6 +242,7 @@ def get_assigned_employees(self, from_date=None, consider_default_shift=False) -
 		assigned_employees = frappe.get_all("Shift Assignment", filters=filters, pluck="employee")
 
 		if consider_default_shift:
+			filters["start_date"] = ("=", from_date)
```

Comment:
This might not be correct. Shifts that start before this date are also applicable as "assigned"